aten/src/ATen/native/cuda/Math.cuh

```diff
@@ -174,6 +174,504 @@ static inline __host__ __device__ scalar_t calc_polygamma(int n, scalar_t x) {
   return ((n % 2) ? 1.0 : -1.0) * ::exp(::lgamma(static_cast<scalar_t>(n) + 1.0)) * zeta(static_cast<scalar_t>(n + 1), x);
 }
 
+/*
+ * The implementation of incomplete gamma function as well as the helper
```

Comment:
"This implementation of the regularized incomplete gamma functions and their helper functions are derived from the implementation of SciPy's gammainc, Cephes's igam and igamc, and Boost's Lanczos approximations. See NOTICE for the licenses."